entries/p/pepper.com.json

```
@@ -11,6 +9,19 @@
     "keywords": [
       "social"
     ],
+    "additional-domains": [
+      "promodescuentos.com",
+      "pelando.com.br",
+      "hotukdeals.com",
+      "dealabs.com",
+      "chollometro.com",
+      "mydealz.de",
+      "pepper.it",
+      "pepper.pl",
+      "preisjaeger.at",
+      "pepper.ru",
+      "desidime.com"
+    ],
```

May sorting these alphabetically is useful:
```suggestion
    "additional-domains": [
      "chollometro.com",
      "dealabs.com",
      "desidime.com",
      "hotukdeals.com",
      "mydealz.de",
      "pelando.com.br",
      "pepper.it",
      "pepper.pl",
      "pepper.ru",
      "preisjaeger.at",
      "promodescuentos.com"
    ],
```